microreactors/gcmr/core/MESH/Griffin_mesh.i

```diff
@@ -0,0 +1,650 @@
+[Mesh]
+  [YH_pin]
+    type = PolygonConcentricCircleMeshGenerator
+    num_sides = 6
+    num_sectors_per_side = '2 2 2 2 2 2'
+    background_intervals = 1
+    background_block_ids = '10'
+    polygon_size = 1
+    polygon_size_style ='apothem'
+    ring_radii = '0.693 0.7 0.75'
+    ring_intervals = '2 1 1'
+    ring_block_ids = '101 100 102 103'
+    preserve_volumes = on
+    #quad_center_elements = true
+    #external_boundary_id = 100
+    #interface_boundary_id_shift = 100
```

Comment:
Remove commented lined, same for other mesh blocks

Reply:
Removed.